hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/rmapp/RMAppImpl.java

```diff
@@ -1058,10 +1058,8 @@ public void transition(RMAppImpl app, RMAppEvent event) {
       // otherwise, add it to ranNodes for further process
       app.ranNodes.add(nodeAddedEvent.getNodeId());
 
-      if (!nodeAddedEvent.getFromAcquiredState()) {
-        app.logAggregation.addReportIfNecessary(
-                nodeAddedEvent.getNodeId(), app.getApplicationId());
-      }
+      app.logAggregation.addReportIfNecessary(
```

Comment:
This doesn't seem to be part of the backport ?
https://issues.apache.org/jira/secure/attachment/13001908/YARN-10244-branch-3.2.003.patch

Any impact of doing it even if event is not from acquired state

Reply:
This is because https://issues.apache.org/jira/browse/YARN-9877
Above patch, we suspected for log aggregation issue so applied and tested but haven't revered from Flipkart branch. In my repo it's revered.